src/screens/Settings/ReportIssue/index.tsx

```diff
@@ -0,0 +1,185 @@
+}: SettingsScreenProps<'ReportIssue'>): ReactElement => {
+	const { t } = useTranslation('settings');
+	const [email, setEmail] = useState('');
+	const [message, setMessage] = useState('');
+	const [isLoading, setIsLoading] = useState(false);
+
+	const validateEmail = (emailText: string): boolean => {
+		if (emailText.indexOf('@') !== -1) {
+			const [beforeAt, afterAt] = emailText.split('@');
+			if (beforeAt.length > 0 && afterAt.length > 0) {
+				return true;
+			}
+		}
+		return false;
+	};
+
+	const sendRequest = async (): Promise<void> => {
+		try {
+			setIsLoading(true);
+			let ldkVersion = '';
+			let ldkNodeId = '';
+			let logs = '';
+			let logsFileName = '';
+			const ldkVersionUser = await getNodeVersion();
+			const ldknodeIdUser = await getNodeId();
+			const logsUser = await zipLogs();
+
+			if (ldkVersionUser.isOk() && ldknodeIdUser.isOk() && logsUser.isOk()) {
+				ldkVersion += `ldk-${ldkVersionUser.value.ldk} c_bindings-${ldkVersionUser.value.c_bindings}`;
+				ldkNodeId += `${ldknodeIdUser.value}`;
+				const logsContent = await RNFS.readFile(logsUser.value, 'base64');
+				const logsName = logsUser.value.match(/\/([^/]+)\.zip$/);
+				logs += `${logsContent}`;
+				if (logsName && logsName.length > 1) {
+					logsFileName += `${logsName[1]}`;
+				}
+			}
+			await axios.post(`${__CHATWOOT_API__}`, {
+				email,
+				message,
+				platform: `${Platform.OS} ${getSystemVersion()}`,
+				version: `${getVersion()} (${getBuildNumber()})`,
+				ldkVersion: ldkVersion,
+				ldkNodeId: ldkNodeId,
+				logs: logs,
+				logsFileName: logsFileName,
+			});
+			navigation.navigate('FormSuccess');
+			setEmail('');
+			setMessage('');
+			setIsLoading(false);
+		} catch (error) {
+			console.error('Error', error);
+			navigation.navigate('FormError');
+			setEmail('');
+			setMessage('');
+			setIsLoading(false);
+		}
+	};
+
+	const isValid = validateEmail(email) && message;
+
+	return (
+		<ThemedView style={styles.fullHeight}>
+			<KeyboardAvoidingView style={styles.content}>
+				<ScrollView
+					color="transparent"
+					contentContainerStyle={styles.scrollContent}
+					showsVerticalScrollIndicator={false}
+					bounces={false}>
+					<SafeAreaInset type="top" />
+					<NavigationHeader
+						title={t('support.report')}
+						onClosePress={(): void => {
+							navigation.navigate('Wallet');
+						}}
+					/>
+					<View style={styles.form}>
+						<Text01S style={styles.text} color="gray1">
+							{t('support.report_text')}
+						</Text01S>
+
+						<LabeledInput
+							style={styles.addressInput}
+							label={t('support.label_address')}
+							placeholder={t('support.placeholder_address')}
+							maxLength={25}
+							value={email}
+							testID="emailValueInput"
+							onChange={(value: string): void => {
+								setEmail(value);
+							}}
+						/>
+						<LabeledInput
+							placeholder={t('support.placeholder_message')}
+							multiline={true}
+							value={message}
+							returnKeyType="done"
```

Comment:
This is not behaving correctly together with `multiline={true}` on iOS. Should be okay to use "default" (line break) since we have scrolling now.

```suggestion
							returnKeyType="default"
```

Reply:
Done